src/main/java/de/bonndan/nivio/api/ApiController.java

```diff
@@ -60,19 +61,22 @@ public ApiController(LandscapeRepository landscapeRepository, ItemDescriptionFor
     private Function<LandscapeImpl, LandscapeDTO> createDTO() {
         return landscape -> {
             LandscapeDTO dto = LandscapeDTOFactory.from(landscape);
+
             LandscapeDTOFactory.addLinks(dto);
             return dto;
         };
     }
 
     @CrossOrigin(methods = RequestMethod.GET)
-    @RequestMapping(path = "/landscape/{identifier}")
+    @RequestMapping(path = "/landscape/{identifier}", produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<LandscapeDTO> landscape(@PathVariable String identifier) {
         LandscapeImpl landscape = landscapeRepository.findDistinctByIdentifier(identifier).orElse(null);
         if (landscape == null)
             return ResponseEntity.notFound().build();
-
-        return new ResponseEntity<>(createDTO().apply(landscape), HttpStatus.OK);
+        Assessment assessment = new Assessment(landscape.getConfig().getKPIs());
+        assessment.assess(landscape);
```

Comment:
I had in mind to decouple assessment (volatile state) from the landscape (much more static). This could be also worth an extra API endpoint (?)

But then again there is the Assessable interface which encourages to set state on landscape components. Any good idea? @Matthimatiker

Reply:
I would need more information to make a qualified decision here.

Separating the static landscape from the state sound reasonable.

Regarding `Assessable`: What if state was returned instead of assigned?

Reply:
changed as proposed. Assessment is a separate thing now with its own API and references to landscape components